corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -1030,13 +1030,8 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
             self.labelWidth = "";
             self.questionTileWidth = `col-md-${columnWidth}`;
         } else {
-            if (self.isLabel || self.isButton) {
-                self.controlWidth = "";
```

Comment:
why was this changed needed?

Reply:
The bug had the case with isButton = true and hasLabel = false (see line1037). Before this change only `offset-md-4` was applied but the width was still 12 causing the column to over flow on the right. After it is `col-md-8 offset-md-4`, moving the column to the right but also shrinking it by 4.